Directory.Packages.props

```diff
@@ -14,6 +14,7 @@
     <PackageVersion Include="LibGit2Sharp" Version="0.27.2" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
     <PackageVersion Include="Microsoft.Toolkit.HighPerformance" Version="7.1.2" />
+    <PackageVersion Include="StrongOf" Version="1.2.4" />
```

Comment:
Do we _really_ need another external dependency?

Reply:
IMO yes, because it is a light-weight, mature cure for Primitive Obsession (refer to https://github.com/BenjaminAbt/StrongOf/blob/main/README.md).